From 699780a12ff532df76ce6b6862a981b39caa4ebc Mon Sep 17 00:00:00 2001
From: juztamau5 <juztamau5@gmail.com>
Date: Sat, 4 Jun 2022 15:20:26 -0700
Subject: [PATCH 1/5] Silence compiler warning

Warning was:

  contracts/depends/sushiswap/SushiRoll.sol:80:9: Warning: Unused function
  parameter. Remove or comment out the variable name to silence this
  warning.
          uint256 deadline
          ^--------------^
---
 contracts/SushiRoll.sol | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contracts/SushiRoll.sol b/contracts/SushiRoll.sol
index 7becfb69..3dd552ca 100644
--- a/contracts/SushiRoll.sol
+++ b/contracts/SushiRoll.sol
@@ -77,7 +77,7 @@ contract SushiRoll {
         uint256 liquidity,
         uint256 amountAMin,
         uint256 amountBMin,
-        uint256 deadline
+        uint256
     ) internal returns (uint256 amountA, uint256 amountB) {
         IUniswapV2Pair pair = IUniswapV2Pair(pairForOldRouter(tokenA, tokenB));
         pair.transferFrom(msg.sender, address(pair), liquidity);
-- 
2.30.2

